kubernetes-discovery-client/src/main/java/io/micronaut/kubernetes/util/KubernetesUtils.java

```diff
@@ -147,7 +147,17 @@ public static PropertySource secretAsPropertySource(Secret secret) {
             if (LOG.isTraceEnabled()) {
                 LOG.trace("Includes: {}", includes);
             }
-            includesFilter = s -> includes.contains(s.getMetadata().getName());
+            includesFilter = s -> {
+                boolean result = includes.contains(s.getMetadata().getName());
```

Comment:
adds more trace logging for include,exclude and labels filters so it's easier to debug